src/app/features/event/event-form/hashtags/hashtags.service.ts

```diff
@@ -0,0 +1,129 @@
+import { ElementRef, inject, Injectable } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import {
+  MatAutocompleteSelectedEvent,
+  MatAutocompleteTrigger,
+} from '@angular/material/autocomplete';
+import { MatChipInputEvent, MatChipEditedEvent } from '@angular/material/chips';
+import { SnackBarService } from '@shared/services';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HashtagsService {
+  private snackBarService = inject(SnackBarService);
+  private readonly MAX_CHARS = 50;
+  private readonly MAX_HASHTAG_COUNT = 100;
+  private addedHashtags: string[] = [];
+  private allHashtags: string[] = [
+    'Impreza',
+    'Piwo',
+    'Piłkarzyki',
+    'Pizza',
+    'Pizza 2',
+    'Pizza 3',
+    'Pizza 4',
+    'Pizza 5',
+    'Karaoke',
+    'Bilard',
+    'Urodziny',
+    'BaaaaaaaaaardzoDłuuuuuugiHashtagAwogóletoooooooooo',
+  ];
```

Comment:
co to za cudeńko?

Reply:
Zmockowane dane do autocomplete, zgodnie z taskiem na ten moment używamy danych zmockowanych w komponencie/serwisie. Jeśli tester będzie chciał to dodam więcej opcji :)